README.md

```diff
@@ -148,7 +148,39 @@ For non-Java applications or tools like `cqlsh`, you can run the Spanner Cassand
 
     This will create an executable jar file `spanner-cassandra-launcher.jar` inside the folder `google-cloud-spanner-cassandra/target`.
 
-* Run the jar using the command:
+* Run the jar using one of the following methods:
+
+    **1. Using a YAML Configuration File (Recommended for Production)**
+
+    For production setups, it is recommended to use a YAML file to configure the adapter. This method supports multiple listeners and global settings. See the [Configuration Options](docs/config-options.md) for a complete list of all supported options.
+
+    Create a `config.yaml` file with the following structure:
+    ```yaml
+    globalClientConfigs:
+      enableBuiltInMetrics: true
+      healthCheckEndpoint: "127.0.0.1:8080"
+    listeners:
+      - name: "listener_1"
+        host: "127.0.0.1"
+        port: 9042
+        spanner:
+          databaseUri: "projects/p/instances/i/databases/d1"
+          numGrpcChannels: 4
+      - name: "listener_2"
+        host: "127.0.0.1"
+        port: 9043
+        spanner:
+          databaseUri: "projects/p/instances/i/databases/d2"
+    ```
```

Comment:
instead of duplicating here, can we add link to https://github.com/googleapis/java-spanner-cassandra/pull/211/files#diff-8a7dfdbcc55ed7c893b182f805614843dbbf8610637773ae00af7224d657b032R40